bundles/core/src/test/resources/list/v2/test-content.json

```diff
@@ -81,6 +95,22 @@
                     "jcr:title": "Page 2",
                     "cq:lastModified": "2016-09-21T16:12:45.000-07:00"
                 }
+            },
+            "page_3": {
+                "jcr:primaryType": "cq:Page",
+                "jcr:content": {
+                    "jcr:primaryType": "cq:PageContent",
+                    "jcr:title": "Page 3",
+                    "sling:vanityPath": "/page_3.html"
+                }
+            },
+            "page_4": {
+                "jcr:primaryType": "cq:Page",
+                "jcr:content": {
+                    "jcr:primaryType": "cq:PageContent",
+                    "jcr:title": "Page 4",
+                    "sling:vanityPath": "/4_page.html"
```

Comment:
Shouldn't it be page_4.html as for page_3.html?

Reply:
No, I specifically named the vanity path to be `4_page`